CONTRIBUTING.md

```diff
@@ -49,15 +49,15 @@ To get started, run `bin/setup`
 
 After setting up, you can run the application using [foreman]:
 
-    % foreman start
+    foreman start
```

Comment:
hey there! I usually use a `%` of `$` to indicate that a command is on the command line (my zsh setup uses `%` as a prompt but I know `$` is more common)

Do you think having this here is confusing? how else do you think we can communicate that a blob of code is for the command line?